Bio/Phylo/_io.py

```diff
@@ -34,7 +34,7 @@
 
 
 def parse(file, format, **kwargs):
-    """Iteratively parse a file and return each of the trees it contains.
+    """Parse a file and return each of the trees it contains.
```

Comment:
It would be good to have something in the one line indicating this is an iterator, maybe "Parse a file iteratively, and yield each of the trees it contains."?